index.html

```diff
@@ -5,8 +5,14 @@
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Adel's React calculator App</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=League+Spartan:wght@700&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body class="">
+  <body class="themeOne font-leagueSpartan" id="themeOne">
     <div class="" id="root"></div>
```

Comment:
Can we remove the empty class? 
```suggestion
    <div id="root"></div>
```